tests/src/Functional/MsSequelServerNativeWebServicesTest.php

```diff
@@ -38,12 +38,6 @@ public function testGeneratedCode()
         // we should consider separating generated code between methods - e.g.
         // by adding the test method as a namespace.
         $this->assertGeneratedClassExists('DayAsNumber');
-
-        $object = new \DayAsNumber(new \DateTime());
-        $class = new \ReflectionClass($object);
-
-        $this->assertMethodParameterHasType($class->getConstructor(), 'day', 'DateTime');
-        $this->assertMethodParameterDocBlockHasType($class->getConstructor(), 'day', '\DateTime');
```

Comment:
Why is this removed?

If these assertions are no longer valid then I would expected the to be replaced with something else.